code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -71,98 +79,67 @@
 	..()
 	can_hold = typecacheof(can_hold)
 
-/obj/item/gripper/attack_self(mob/user as mob)
-	if(wrapped)
-		wrapped.attack_self(user)
-
-/obj/item/gripper/verb/drop_item()
-
-	set name = "Drop Gripped Item"
-	set desc = "Release an item from your magnetic gripper."
-	set category = "Drone"
-
-	drop_item_p()
-
-// The "p" stands for proc, since I was having annoying weird stuff happening with this in the verb
-// when trying to have default values for arguments and stuff
-/obj/item/gripper/proc/drop_item_p(var/silent = 0)
-
-	if(!wrapped)
-		//There's some weirdness with items being lost inside the arm. Trying to fix all cases. ~Z
-		for(var/obj/item/thing in src.contents)
-			thing.forceMove(get_turf(src))
-		return
-
-	if(wrapped.loc != src)
-		wrapped = null
-		return
+/obj/item/gripper/attack_self(mob/user)
+	if(gripped_item)
+		if(gripped_item.type in configurable_items)
+			var/choice = alert(user, "Do you want to configure/place the item, or drop it?", "Gripper selection", "Configure/place", "Drop", "Cancel")
+			switch(choice)
+				if("Configure/place")
+					gripped_item.attack_self(user)
+				if("Drop")
+					drop_gripped_item()
+				if("Cancel")
+					return FALSE
+		else
+			drop_gripped_item()
+	else
+		to_chat(user, "<span class='warning'>Your gripper is empty.</span>")
 
+/obj/item/gripper/proc/drop_gripped_item(silent = 0)
 	if(!silent)
-		to_chat(src.loc, "<span class='warning'>You drop \the [wrapped].</span>")
-	wrapped.forceMove(get_turf(src))
-	wrapped = null
+		to_chat(src.loc, "<span class='warning'>You drop the [gripped_item].</span>")
```

Comment:
"the" should also not be needed I think

Reply:
Whats annoying is that for some items like airlock electronics, it will say "You drop the airlock electronics" which is fine. But for other items like cells it will say "You drop the the high capacity cell". So in one instance its fine to leave it, but if the item already has a "the" attached to the front, it makes it look ugly.

Reply:
yeah I got no clue how the game figures that out tbh